desktop-src/menurc/using-menus.md

```diff
@@ -99,7 +99,7 @@ The following example shows how to create a class menu for an application, creat
 Following is the relevant portion of the application's header file:
 
 
-```
+```C++
```

Comment:
Should be ```cpp``

Reply:
Is it a new convention?
Because before I made my changes, I specifically looked up for that in multiple files, and found that the ```C++ is the accepted convention everywhere.

e.g. (just to name a few):
https://github.com/MicrosoftDocs/win32/blame/docs/desktop-src/LearnWin32/accelerator-tables.md#L51
https://github.com/MicrosoftDocs/win32/blame/docs/desktop-src/Direct2D/3d-transform.md#L44

Reply:
IMO the language should be (pure) C, but never mind, that was just a nit. Thanks

Reply:
The preferred way to indicate the language is to use the alias `cpp`. See the [list of language aliases is the Docs Authoring Guide](https://docs.microsoft.com/contribute/code-in-docs#snippet-syntax-reference). If you use a supported alias, the doc build will transform it to the appropriate language name. Otherwise, the language name text it displayed as-is.